### ISSUE TITLE
Question: is there a way to invalidate only some urls?

### ISSUE THREAD

**Issue by tim-moody** [darkenvy/mwoffliner-article-cacher#1]
If the articles that have changed could be identified is there a way to invalidate the cache only for those articles?

**Comment by tim-moody** [comment]
would it work to run a url through the hash function and then remove the resultant file name?

**Comment by psiie** [comment]
Yes that should work! The cache doesn't have a db, it just looks for a file on the hard-drive. (more performance could be had with a db, but it seemed like over-engineering). If you delete the corresponding hash it should force it to get a new one. 

PS: The only reason for the hash was to resolve saving urls as filename. (It might also benefit being a hash for filename lookup too.)
